telegram/message.py

```diff
@@ -436,6 +490,8 @@ def __init__(
         self.reply_markup = reply_markup
         self.bot = bot
 
+        self._effective_attachment = _UNDEFINED
```

Comment:
We would use `DEFAULT_NONE` here …

Reply:
Yea? I just moved that as from a class variable to instance variable. It says the same on master too. Anyway will change

Reply:
No I mean since the introduction of `DefaultValue` (which was introduced after `Message.effective_attachment`), we have the package-wide constant `DEFAULT_NONE` to denote "not set, but also not equivalent to `None`". So I suggest to use it here as well instead of a module-specific solution :)